CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.9 - TO BE RELEASED
+
+### Fixed
+- Fixed deprecation notice for LogoutSuccessHandler by replacing it with LogoutEventListener
+
```

Comment:
I'm not sure if we should highlight every internal technical debt change / fixes in the changelog, it doesn't (shouldn't) affect the behavior of the application. What do you think?

Reply:
:+1: 
Indeed, in such case it's not really needed. Would remove it